app/konnect/gateway-manager/provision-cloud-gateway.md

```diff
@@ -0,0 +1,280 @@
+		"control_plane_id": "3e62d2c5-45e9-4046-8065-9cd28cb3487d",
+		"version": "3.6",
+		"control_plane_geo": "us",
+		"dataplane_groups": [
+			{
+				"provider": "aws",
+				"region": "ap-northeast-1",
+				"cloud_gateway_network_id": "0e465f2f-4fa3-43ed-8900-bb6210c1e554",
+				"autoscale": {
+					"kind": "autopilot",
+					"base_rps": 100
+				}
+			}
+		]
+	}'
+    ```
+	Replace the PAT as well as the following placeholders with your own values:
+	* `control_plane_id`: The ID of the control plane you created in the previous step.
+	* `version`: The version of {{site.base_gateway}} you want to use.
+	* `control_plane_geo`: The geo of the control plane.
+	* `region`: The AWS region you want to deploy the data plane nodes in.
+	* `cloud_gateway_network_id`: The network ID of the cloud gateway.
+	* `kind`: The configuration mode. Choose between `static` or `autopilot`. Autopilot mode allows Kong to automatically scale your instances based on incoming traffic. You can pre-warm your cluster by specifying the number of requests per second. Custom mode (`static`) allows you to select from three different instance sizes: small, medium, or large.
+
+    You should get a `201` response like the following:
+
+    ```sh
+    {
+	"id": "3682c148-1985-44a0-8cf0-c49f76a53dda",
+	"control_plane_id": "3e62d2e6-45e9-4046-8065-9cd28cb3487d",
+	"control_plane_geo": "us",
+	"version": "3.6",
+	"api_access": "private+public",
+	"dataplane_group_config": [
+		{
+		"provider": "aws",
+		"region": "ap-northeast-1",
+		"autoscale": {
+			"kind": "autopilot",
+			"base_rps": 100,
+			"max_rps": 1000
+		},
+		"cloud_gateway_network_id": "0e465f2f-4fa3-43ed-8900-bb6210c1e554"
+		}
+	],
+	"dataplane_groups": [
+		{
+		"id": "a27b8b9b-f3c9-42f6-b0ea-5a678c008298",
+		"provider": "aws",
+		"region": "ap-northeast-1",
+		"autoscale": {
+			"kind": "autopilot",
+			"base_rps": 100,
+			"max_rps": 1000
+		},
+		"state": "initializing",
+		"cloud_gateway_network_id": "0e465f2f-4fa3-43ed-8900-bb6210c1e554",
+		"created_at": "2024-04-04T14:14:22.717Z",
+		"updated_at": "2024-04-04T14:14:22.717Z"
+		}
+	],
+	"created_at": "2024-04-04T14:14:22.717Z",
+	"updated_at": "2024-04-04T14:14:22.717Z",
+	"entity_version": 1
+    }
+    ```
+
+Your cloud gateway is now provisioned. You can use it like you would any other {{site.base_gateway}} in {{site.konnect_short_name}}.
+{% endnavtab %}
+{% endnavtabs %}
+
+## Upgrade your fully-managed data plane nodes
+
+{{site.konnect_short_name}} handles upgrades for you. There's no downtime when upgrading the infrastructure. For more information about how to upgrade your data plane nodes, see [Upgrade a Data Plane Node to a New Version](/konnect/gateway-manager/data-plane-nodes/upgrade/)
+
+## Scale your fully-managed data plane nodes
+
+{% navtabs %}
+{% navtab UI %}
+1. From [Gateway Manager](https://cloud.konghq.com/gateway-manager) in the navigation menu, click the Dedicated Cloud Gateway control plane you want to scale the data plane nodes for.
```

Comment:
```suggestion
1. From [Gateway Manager](https://cloud.konghq.com/gateway-manager), click the Dedicated Cloud Gateway control plane you want to scale the data plane nodes for.
```
I'm not sure what this instance of "in the navigation menu" is trying to tell me. My first instinct is to look for a list of nodes in a menu somewhere, but they're listed directly on the page.